lesson10/index.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Interfaces,Abstarct and Final classes
+nav_order: 10
+has_children: true
+---
+
+## Learning Objectives
+
+At the end of the class, attendees we be able to
+
+- Define what an final class is
+- Create and use final classes in Java
+- Explain what final classes are useful for
```

Comment:
We've covered final before - so they might know a little bit about it. :-) Anyways good to talk about it again more formally.